handler.js

```diff
@@ -43,6 +43,14 @@ buildUsers = buildUsers != "" ? buildUsers.split(",") : [];
 var buildComment = getObjectDefault(process.env, "GITHUB_BUILD_COMMENT", "");
 buildComment = buildComment != "" ? buildComment : "go codebuild go";
 
+// if set to true, CodeBuild will not include PR number in request from buildspec from git
```

Comment:
"Controls whether the `sourceVersion` parameter is set to the pull request and passed to the CodeBuild job. Set to true if the repo for the CodeBuild project itself is different from the repo where the webhook will be created"